hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -127,13 +133,41 @@ public double getRatio() {
     }
 
     @Override
+    public long getEvictions() {
+        return evictions;
+    }
+
+    public void setEvictions(long evictions) {
```

Comment:
It's just used in a test scenario. The value is just incremented during runtime.